docs/usage.md

```diff
@@ -38,9 +38,22 @@ The following matrices are displayed by the SOC Management Widget:
   - **Playbook Run** displays the total number of playbooks executed in the specified time range.
   - **Actions Executed** displays the total number of playbook actions executed in the specified time range.
   - **ROI** displays the ROI of playbook actions executed in terms of savings in the dollars in the specified time range. 
+    While calculating ROI it is considered, to manually make API call it will take 5 minutes and an analyst would charge 60$ per hour.
+    For example, if the widget is configured for 1 day, and the actions executed in the day are 10 then the ROI would be $50.
   - **Alert Resolved** displays the total number of alerts resolved in the specified time range.
   - **Overall Time Saved** displays the time saved by executing playbook actions in the specified time range. 
+    Overall time saved is calculated by multiplying the actions executed count by 5 minutes,  i.e. the time taken by an analyst to manually do the task. The value is displayed in hrs. 
+    eg. if the actions executed are 97, the overall time saved will be (97x5)/60 = 8Hrs

```

Comment:
This is good addition, info. Can we create a table as well to show these calculations all together with all possible combinations?